blocks/hotspot/hotspot.css

```diff
@@ -0,0 +1,41 @@
+.block.hotspot {
+    position: relative;
+}
+
+.block.hotspot.right {
+   right: -15rem;
+   top: 0;
+}
+
+.block.hotspot img {
+    max-width: 100%;
+    width: auto;
+    height: auto;
+}
+
+
+.block.hotspot > div:nth-child(n+2) {
+    border-radius: 5px;
+    background: white;
+    position: absolute;
+    width: 25vw;
+    cursor: pointer;
```

Comment:
The original page has `cursor:text`, I think either this or just the regular cursor is better since you can't click on this text